packages/mds-db/geographies.ts

```diff
@@ -82,6 +87,36 @@ export async function readGeographies(params: Partial<ReadGeographiesParams> = {
   }
 }
 
+export async function readPublishedGeographies(only_published_after?: Timestamp): Promise<Geography[]> {
+  try {
+    const client = await getReadOnlyClient()
+
+    let sql = `SELECT * FROM ${schema.TABLE.geographies}`
+
+    let conditions = ''
+    const vals = new SqlVals()
+
+    if (only_published_after) {
+      conditions = `publish_date > ${only_published_after}`
+    } else {
+      conditions = `publish_date IS NOT NULL`
+    }
+
+    sql += ` WHERE ${conditions}`
+
+    const values = vals.values()
+    const { rows } = await client.query(sql, values)
```

Comment:
We don't really need `SqlVals` here I think. That's to protect against SQL injection, which isn't possible in this case.

Reply:
Actually, I think SQL injection is still a possibility here. The type of `only_published_after` might be `Timestamp` in the code, but that doesn't give any runtime guarantees. This should be changed to `conditions = `publish_date > ${vals.add(only_published_after)}`

Reply:
thanks!